scripts/update-ruby.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
```

Comment:
This needs FB copyright header comment, you can copy from the other .sh file in the same dir:

```

# Copyright (c) Facebook, Inc. and its affiliates.
#
# This source code is licensed under the MIT license found in the
# LICENSE file in the root directory of this source tree.
```

Reply:
@fkgozali ok, can I add my company name or it must be assigned to FB?

Reply:
Because the file lives within FB OSS repo, this needs to be under the same FB copyright header